app/models/manageiq/providers/ibm_terraform/provider.rb

```diff
@@ -62,6 +66,15 @@ def self.params_for_create
                   :isRequired => true,
                   :validate   => [{:type => "required"}]
                 },
+                {
+                  :component  => "text-field",
+                  :name       => "endpoints.cpd.url",
+                  :id         => "endpoints.cpd.url",
+                  :label      => _("IBM Cloud Pak url"),
```

Comment:
```suggestion
                  :label      => _("IBM Cloud Pak URL"),
```

Reply:
Is this different from the `IBM Cloud Pak console URL. e.g. https://cp-console.apps.mydomain.com"` ?  Do we need both?

Reply:
We need both the urls IBM Cloud Pak url will be used for accessing UI and Cloud Pak console URL for auethntication

Reply:
But do we need 3? We already have:
1. A default endpoint: `Managed Services URL. e.g. https://cam.apps.mydomain.com`
2. An identity/console endpoint: `IBM Cloud Pak console URL. e.g. https://cp-console.apps.mydomain.com`
3. And now a CPD endpoint: `IBM Cloud Pak URL. e.g. https://cpd-cp4waiops.apps.mydomain.com`

Reply:
The answer might be yes @mahantidinesh7 I'm just wondering if we should be replacing one rather than adding a new one.  Are all three actively used?

Also wondering if we should be taking a "base" url and building service urls off of that.  Are `cam.`, `cp-consoe.` or `cpd-cp4waiops.` configurable?

Reply:
Yeah, we discussed downstream - I'll ping you in the discussion